ethan-aws-finops-culture-blog

```diff
@@ -0,0 +1,86 @@
+//Ethan AWS Cost Optimisation Blog on FinOps Culture 
+
+**Looking Beyond The Technology in FinOps - How organisations need to address the People Side **
+
+FinOps, or cloud financial management, has grown significantly over the past few years, with more and more organisations prioritising it to reap critical benefits, including saving money and driving better accountability across their IT estate. To meet demand or to focus on quick wins early, organisations have steered towards the technology or tooling side first without adopting the people or process element, which drives long-term success. 
```

Comment:
add to the start more of a grabbing intro of why someone should read this blog and who it is for and what problem you are sloving